cfme/tests/cloud/test_stack.py

```diff
@@ -0,0 +1,59 @@
+import pytest
+from cfme.fixtures import pytest_selenium as sel
+from cfme.cloud.stack import Stack
+from utils import testgen
+from utils.providers import setup_provider
+from cfme.configure import settings
+from cfme.web_ui import ButtonGroup, form_buttons
+
+
+pytestmark = [
+    pytest.mark.usefixtures('logged_in'),
+    pytest.mark.ignore_stream("5.3"),
+    pytest.mark.ignore_stream("upstream")
+]
+
+pytest_generate_tests = testgen.generate(testgen.provider_by_type,
+    provider_types=['ec2'], scope="module")
+
+
+@pytest.fixture
+def provider_init(provider_key):
+    """cfme/cloud/provider.py provider object."""
+    setup_provider(provider_key)
+
+
+def set_grid_view(name):
+    bg = ButtonGroup(name)
+    sel.force_navigate("my_settings_default_views")
+    default_view = bg.active
+    if(default_view != 'Grid View'):
+        bg.choose('Grid View')
+        sel.click(form_buttons.save)
+
+
+def test_stack_detail_page(provider_init):
+    set_grid_view("Stacks")
+    stack = Stack(name="newstack")
```

Comment:
Right now , we can't create stack from CFME . I created this stack "newstack" in aws and hardcoded it for now